inspire/base/static/js/search/form.js

```diff
@@ -32,6 +32,10 @@ define([
     return true; // ensure form still submits
   })
 
+  $("button[name=action_search]").on("click", function(){
```

Comment:
When you add a search query to the search bar and press Enter, does it count as `click` ?